Windows build step is broken

https://circleci.com/gh/timberio/vector/94082?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
   Compiling vector v0.9.0 (C:\Users\circleci\project)
   Compiling rdkafka v0.23.1
error[E0425]: cannot find value `metrics_trigger` in this scope
   --> src\main.rs:388:9
    |
388 |         metrics_trigger.cancel();
    |         ^^^^^^^^^^^^^^^ not found in this scope

error: aborting due to previous error

For more information about this error, try `rustc --explain E0425`.
error: could not compile `vector`.
```

@Hoverbear @a-rodin any idea what's going on here? It's blocking `0.9.0`, which can wait until tomorrow.

--- a/src/main.rs
+++ b/src/main.rs
@@ -385,7 +385,6 @@ fn main() {
 
         info!("Shutting down.");
         let shutdown = topology.stop();
-        metrics_trigger.cancel();
 
         match rt.block_on(shutdown.select2(ctrl_c)) {
             Ok(Either::A(_)) => { /* Graceful shutdown finished */ }

